pkg/storj/object.go

```diff
@@ -0,0 +1,68 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package storj
+
+import "time"
+
+// Bucket contains information about a specific bucket
+type Bucket struct {
+	Name    string
+	Created time.Time
+}
+
+// Object contains information about a specific object
+type Object struct {
+	Version  uint32
+	Bucket   string
+	Path     Path
+	IsPrefix bool
+
+	Metadata []byte
+
+	ContentType string
+	Created     time.Time
+	Modified    time.Time
+	Expires     time.Time
+
+	Stream
+}
+
+// Stream is information about an object stream
+type Stream struct {
+	// Size is the total size of the stream in bytes
+	Size int64
+
```

Comment:
Should we have some `Checksum` field here too? We used to have such filed in V2.

Reply:
Sure.